.github/workflows/enso4igv.yml

```diff
@@ -24,6 +25,14 @@ jobs:
           java-version: ${{ matrix.java }}
           cache: maven
 
+      - uses: graalvm/setup-graalvm@v1
+        with:
+          java-version: '21'
+          distribution: 'graalvm-community'
+
+      - name: Publish to Maven Repository
+        run: sbt publishM2
```

Comment:
This is useful in preparation for
- #7054

but it also verifies the Javadoc of all published modules (currently mostly `persitance` API) is valid without _any broken references_. Which [it is](https://github.com/enso-org/enso/actions/runs/9436278545/job/25990770918?pr=10222#step:5:157).